pyproject.toml

```
@@ -60,7 +59,7 @@ pytest = "^6.2"
 pytest-cov = "^2.8"
 pytest-mock = "^3.5"
 pytest-sugar = "^0.9"
-pre-commit = { version = "^2.6", python = "^3.6.1" }
+pre-commit = { version = "^2.6", python = "^3.7" }
```

the `python` marker is no longer necessary
```suggestion
pre-commit = "^2.6"
```